.github/workflows/ci.yml

```diff
@@ -33,6 +36,9 @@ jobs:
           uv run ruff check
           uv run ruff format
 
+      - name: Run Pyright
+        run: pyright
```

Comment:
I think `uv run pyright` is better here for the reason explained above

Reply:
You're absolutely right. I'll make the changes.